src/threading.ts

```diff
@@ -247,11 +270,18 @@ export class BytecodeStackFrame implements IStackFrame {
 
     // Run until we get the signal to return to the thread loop.
     while (!this.returnToThreadLoop) {
-      var op = code.readUInt8(this.pc);
-      if (!RELEASE && logging.log_level === logging.VTRACE) {
-        vtrace(`  ${this.pc} ${annotateOpcode(op, this, code, this.pc)}`);
+      var op = method.getOp(this.pc, code);
```

Comment:
I just had another idea: count function executions + jumps with negative offsets. It is simple enough to not have an overhead, and specific enough to improve JIT targets.

Will try it out and push to this PR if it improves performance.